app/src/main/java/com/example/soundcloud/selected_genre_detail/SelectedGenreActivity.java

```diff
@@ -16,6 +16,8 @@
     private TextView mTextViewGenreTitle;
     private ImageButton mImageButtonSearch;
     private RecyclerView mRecyclerView;
```

Comment:
RecyclerView thì đặt tên cho có ý nghĩa hơn
```suggestion
    private RecyclerView mRecyclerGenres;
```